lib/pbcs/aes_cbc_hmac_sha2.ex

```diff
@@ -9,8 +9,8 @@ defmodule PBCS.AES_CBC_HMAC_SHA2 do
   See: https://tools.ietf.org/html/rfc7518#section-5.2.6
   """
 
-  @spec content_encrypt({binary, binary}, <<_::32>> | <<_::48>> | <<_::64>>, <<_::16>>) ::
```

Comment:
These were the warnings that I got:

```
lib/pbcs/aes_gcm.ex:13:invalid_contract
Invalid type specification for function.

Function:
PBCS.AES_GCM.content_encrypt/3

Success typing:
@spec content_encrypt({binary(), binary()}, binary(), binary()) :: {binary(), binary()}
________________________________________________________________________________
lib/pbcs/aes_gcm.ex:21:invalid_contract
Invalid type specification for function.

Function:
PBCS.AES_GCM.content_decrypt/3

Success typing:
@spec content_decrypt({binary(), binary(), binary()}, binary(), binary()) :: :error | {:ok, binary()}
```

It looks like I may have over-fixed since the issue was in aes_gcm.ex. I made the ones in aes_cbc_hmac_sha2.ex consistent.

Reply:
Okay 👍